kong/templates/kong_defaults.lua

```diff
@@ -19,6 +19,8 @@ admin_ssl = on
 admin_ssl_cert = NONE
 admin_ssl_cert_key = NONE
 upstream_keepalive = 60
+server_tokens = on
+latency_tokens = on
 
```

Comment:
those also need to be added to the `kong/kong.conf.default` file including a description.